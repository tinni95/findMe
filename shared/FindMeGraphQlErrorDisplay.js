import React from "react";
import { StyleSheet, View } from "react-native";
import { Icon, Text } from "native-base";

const FindMeGraphQlErrorDisplay = () => {
  return (
    <View style={styles.view}>
      <Icon name="alert" style={styles.icon} />
      <Text style={styles.text}>
        {"Oops, something went wrong!\n" +
          "\n" +
          "If this problem persists, please reach out to us by email at diversesoftwaredesign@gmail.com"}
      </Text>
    </View>
  );
};

const styles = StyleSheet.create({
  view: {
    flex: 1,
    flexDirection: "column",
    justifyContent: "center",
    alignItems: "center"
  },
  icon: {
    color: "blue"
  },
  text: {
    color: "black",
    textAlign: "center"
  }
});

export default FindMeGraphQlErrorDisplay;
