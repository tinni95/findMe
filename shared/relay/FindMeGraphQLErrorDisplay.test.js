import React from "react";
import { render } from "react-native-testing-library";

import FindMeGraphQlErrorDisplay from "./FindMeGraphQlErrorDisplay";

describe("FindMeGraphQlErrorDisplay", () => {
    describe("with no non-default props", () => {
        it("renders successfully to match snapshot", () => {
            const rendered = render(<FindMeGraphQlErrorDisplay />);
            expect(rendered.toJSON()).toBeTruthy();
            expect(rendered.toJSON()).toMatchSnapshot();
        });
    });
});
