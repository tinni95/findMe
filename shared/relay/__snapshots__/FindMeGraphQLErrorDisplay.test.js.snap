// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FindMeGraphQlErrorDisplay with no non-default props renders successfully to match snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
>
  <Text />
  <Text
    style={
      Array [
        Object {
          "color": "#000",
          "fontFamily": "System",
          "fontSize": 16,
        },
        Object {
          "color": "black",
          "textAlign": "center",
        },
      ]
    }
    uppercase={false}
  >
    Oops, something went wrong!

If this problem persists, please reach out to us by email at hello@purplesquid.scot
  </Text>
</View>
`;
